docs/src/man/basics.md

```diff
@@ -0,0 +1,1220 @@
+    6 │    35      5      1  free     NA               NA
+    7 │    53      6      2  own      quite rich       NA
+    8 │    35      7      3  rent     little           moderate
+  ⋮   │   ⋮      ⋮      ⋮       ⋮            ⋮                ⋮                ⋱
+  994 │    30    993      3  own      little           little                  ⋯
+  995 │    50    994      2  own      NA               NA
+  996 │    31    995      1  own      little           NA
+  997 │    40    996      3  own      little           little
+  998 │    38    997      2  own      little           NA                      ⋯
+  999 │    23    998      2  free     little           little
+ 1000 │    27    999      2  own      moderate         moderate
+                                                  3 columns and 985 rows omitted
+```
+
+You can also use `Regex` to select columns and `Not` from `InvertedIndices.jl` both to select rows and columns:
+```jldoctest dataframe
+julia> german[Not(5), r"Sex"]
+999×1 DataFrame
+ Row │ Sex
+     │ String
+─────┼────────
+   1 │ male
+   2 │ female
+   3 │ male
+   4 │ male
+   5 │ male
+   6 │ male
+   7 │ male
+   8 │ male
+  ⋮  │   ⋮
+ 993 │ male
+ 994 │ male
+ 995 │ female
+ 996 │ male
+ 997 │ male
+ 998 │ male
+ 999 │ male
+984 rows omitted
+```
+
+`german[!, Not(Age)]` ! indicates that underlying columns are not copied:
+```jldoctest dataframe
+julia> german[!, Not(3)]
+1000×9 DataFrame
+  Row │ id     Age    Job    Housing  Saving accounts  Checking account  Credi ⋯
+      │ Int64  Int64  Int64  String   String           String            Int64 ⋯
+──────┼─────────────────────────────────────────────────────────────────────────
+    1 │     0     67      2  own      NA               little                  ⋯
+    2 │     1     22      2  own      little           moderate
+    3 │     2     49      1  own      little           NA
+    4 │     3     45      2  free     little           little
+    5 │     4     53      2  free     little           little                  ⋯
+    6 │     5     35      1  free     NA               NA
+    7 │     6     53      2  own      quite rich       NA
+    8 │     7     35      3  rent     little           moderate
+  ⋮   │   ⋮      ⋮      ⋮       ⋮            ⋮                ⋮                ⋱
+  994 │   993     30      3  own      little           little                  ⋯
+  995 │   994     50      2  own      NA               NA
+  996 │   995     31      1  own      little           NA
+  997 │   996     40      3  own      little           little
+  998 │   997     38      2  own      little           NA                      ⋯
+  999 │   998     23      2  free     little           little
+ 1000 │   999     27      2  own      moderate         moderate
+                                                  3 columns and 985 rows omitted
+```
+
+In the given code of block `:` means that the columns will get copied:
```

Comment:
why do you repeat the same thing for a third time (maybe it is OK, but please justify)

Reply:
I had discussed difference between `:` and `!`